openpype/hosts/fusion/deploy/openpype/fusion_shared.prefs

```diff
@@ -3,9 +3,9 @@ Locked = true,
 Global = {
   Paths = {
     Map = {
-      ["OpenPype:"] = "$(OPENPYPE_FUSION)/deploy",
+      ["OpenPype:"] = "$(OPENPYPE_FUSION)/deploy/openpype",
       ["Config:"] = "UserPaths:Config;OpenPype:Config",
-      ["Scripts:"] = "UserPaths:Scripts;Reactor:System/Scripts",
+      ["Scripts:"] = "UserPaths:Scripts;Reactor:System/Scripts;OpenPype:../Script",
```

Comment:
This can be removed - openpype needs not to be defined there. There is no `Script` folder.

Reply:
```suggestion
      ["Scripts:"] = "UserPaths:Scripts;Reactor:System/Scripts",
```

Reply:
In that case I am not sure why are we having `openpype\hosts\fusion\deploy\Scripts` and `openpype\hosts\fusion\deploy\Script` folders?

Reply:
I believe those folders could currently be removed - yes. @EmberLightVFX do you know?

Reply:
I don't see those folders in the last OP/Ayon release. I would say they are safe to remove in this PR

Reply:
you are right, those are actually created by Fusion durig launch.

Reply:
> you are right, those are actually created by Fusion during launch.

Do we need to worry about this regarding OpenPype's `checksums`? or does it skip empty folders?